app/controllers/profiles_controller.rb

```diff
@@ -0,0 +1,7 @@
+class ProfilesController < ApplicationController
```

Comment:
And after talking with Chris, I've learned that oc-id needs to map to /id/profile/whatever so I am going to revert everything to how it was in the PR because that seems to be most correct.